# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
bool contact
bool all_sent
